frontend/src/components/beta/transactions/TransactionHeader.tsx

```diff
@@ -0,0 +1,164 @@
+import * as React from "react";
+import { useTranslation } from "react-i18next";
+import { styled } from "../../../libraries/styles";
+import { shortenString } from "../../../libraries/formatting";
+import Moment from "../../../libraries/moment";
+
+import CopyToClipboard from "../../beta/common/CopyToClipboard";
+import { NearAmount } from "../../utils/NearAmount";
+import TransactionStatus from "./TransactionStatus";
+import { TransactionDetails } from "../../../types/common";
+import UtcLabel from "../common/UtcLabel";
+
+type Props = {
+  transaction: TransactionDetails;
+};
+
+const Wrapper = styled("div", {
+  display: "flex",
+  justifyContent: "space-between",
+  width: "100%",
+  backgroundColor: "#1b1d1f",
+  padding: 28,
+  borderRadius: 8,
+  fontFamily: "Manrope",
+});
+
+const BaseInfo = styled("div", {
+  display: "flex",
+});
+
+const AuthorInfo = styled("div", {
+  display: "flex",
+  alignItems: "center",
+});
+
+const Author = styled("div", {
+  display: "flex",
+  alignItems: "center",
+  color: "#fff",
+  fontWeight: 700,
+  fontSize: 30,
+  lineHeight: "150%",
+});
+const Avatar = styled("div", {
+  size: 26,
+  backgroundColor: "#c4c4c4",
+  opacity: 0.2,
+  borderRadius: "50%",
+  marginRight: 8,
+});
+
+const Divider = styled("img", {
+  width: 17,
+  marginHorizontal: 20,
+});
+
+const TransactionHash = styled("h1", {
+  fontFamily: "SF Pro Display",
+  fontSize: 18,
+  fontWeight: 400,
+  color: "#fff",
+  lineHeight: "21px",
+  marginRight: 14,
+  marginLeft: 34,
```

Comment:
I believe this number is tied up to avatar size.
Should we do something like this?
```ts
const AVATAR_SIZE = 26; // any number

const Avatar = styled("div", {
  // ...
  size: AVATAR_SIZE
});

const TransactionHash = stlyed("h1", {
  paddingHorizontal: 14,
  marginLeft: AVATAR_SIZE
})
```